This patch change calling of v to $EDITOR, see https://github.com/DanielFGray/fzf-scripts/issues/26

And add more context to the search.

Also open a file on the specific line

diff --git a/igr b/igr
index 73a6e8c..e23082a 100755
--- a/igr
+++ b/igr
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
-declare preview='bat --color=always --style=header,numbers -H {2} {1} | grep -C3 {q}'
+declare preview='bat --color=always --style=header,numbers -H {2} {1} | grep --context 999 {q}'
 
 while getopts ':l' x; do
   case "$x" in
     l) list_files=1
-      preview='bat --color=always --style=header,numbers {1} | grep -C3 {q}'
+      preview='bat --color=always --style=header,numbers {1} | grep --context 999 {q}'
       ;;
   esac
 done
@@ -18,5 +18,7 @@ fzf -d: \
 --query="$1" \
 --phony \
 --bind="change:reload:rg -n ${list_files:+-l} --color=always {q}" \
---bind='enter:execute:v {1}' \
---preview="[[ -n {1} ]] && $preview"
+--bind='enter:execute:echo "$EDITOR {1}:{2}" && $EDITOR {1}:{2}' \
+--preview="[[ -n {1} ]] && $preview" \
+--preview-window="/2"
+
+ # echo to save in shell history
